### PR TITLE
bomutils 0.2 (new formula)

### PR DESCRIPTION
**Pull request by otaran**

### CODE REVIEW

File: Library/Formula/bomutils.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+  head "https://github.com/hogliux/bomutils.git"
+
+  def install
+    args = %W[
+      CXX=#{ENV.cxx}
+      PREFIX=#{prefix}
+    ]
+
+    system "make", "install", *args
```
Comment (mistydemeo) [inline Library/Formula/bomutils.rb]: If the arguments array isn't being modified, they can just be passed inline.

Comment (otaran) [inline Library/Formula/bomutils.rb]: Right, sorry, I am completely unfamiliar with Ruby.

Comment (mistydemeo) [inline Library/Formula/bomutils.rb]: No worries!

**Comment by mistydemeo** [comment]
Is it possible to write a `test do` block that would test the functionality of this software? You can find some documentation about it [here](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md#add-a-test-to-the-formula).

**Comment by otaran** [comment]
No, I don't think it can be easily tested, because what it basically does is packing directory to binary file and printing out metadata.

**Comment by mikemcquaid** [comment]
It looks like you could follow the tutorial in the `test do` block: http://hogliux.github.io/bomutils/tutorial.html

We need some sort of `test do` for this to be included.

**Comment by DomT4** [comment]
Any update here?

**Comment by DomT4** [comment]
Closing per ~15 days of no response. If you're still interested please address all comments above and open a new PR - Thanks for the submission!

**Pull request closed without merging**